content/docs/guides/blackbox-exporter.md

```diff
@@ -0,0 +1,151 @@
+
+```python
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/health')
+def health():
+    return '', 200
+
+if __name__ == '__main__':
+    app.run(port=2112)
+```
+
+To start up the server:
+
+```shell
+python server.py
+```
+
+You should see something like this in the logs:
+
+```
+Starting up the server on port 2112
+ * Serving Flask app "server" (lazy loading)
+ * Environment: production
+   WARNING: Do not use the development server in a production environment.
+   Use a production WSGI server instead.
+ * Debug mode: off
+ * Running on http://127.0.0.1:2112/ (Press CTRL+C to quit)
+```
+
+To test the `/health` endpoint:
+
+```shell
+curl -i http://localhost:2112/health
+```
+
+If you get a `200 OK` response then the HTTP service is ready to go.
+
+# Installing, configuring, and running Prometheus
+
+To install Prometheus, [download the latest release](/download#prometheus) for your platform and untar it:
+
+```bash
+wget https://github.com/prometheus/prometheus/releases/download/v*/prometheus-*.*-amd64.tar.gz
+tar xvfz prometheus-*.*-amd64.tar.gz
+cd prometheus-*.*-amd64
+```
+
+There is a `prometheus.yml` config file in that directory that you can modify to make Prometheus scrape metrics from the Blackbox Exporter. Replace the contents of that file with this:
+
+```yaml
+scrape_configs:
+- job_name: blackbox
+  metrics_path: /probe
+  scrape_interval: 5s
+  params:
+    module: [http_2xx]
+  static_configs:
+  - targets:
+    - localhost:2112/health
+  relabel_configs:
+  - source_labels: [__address__]
+    target_label: __param_target
+  - source_labels: [__param_target]
+    target_label: instance
+  - target_label: __address__
+    replacement: localhost:9115
+```
+
+This instructs Prometheus to scrape the Blackbox Exporter's `/probe` endpoint and passes the `targets` list, which includes the Python web server running on port 2112, to the Blackbox Exporter. With this configuration in place, start Prometheus:
+
+```shell
+./prometheus
+```
+
+## Installing, configuring, and running the Blackbox Exporter
+
+Like Prometheus, the Blackbox Exporter is a single static binary that you can install via tarball. [Download the latest release](/download#blackbox_exporter) for your platform and untar it:
+
+```bash
+wget https://github.com/prometheus/blackbox_exporter/releases/download/v*/blackbox_exporter-*.*-amd64.tar.gz
+tar xvfz blackbox_exporter-*.*-amd64.tar.gz
+cd blackbox_exporter-*.*-amd64
+```
+
+The untarred folder contains a `blackbox.yml` file for configuring the exporter. Delete the current contents and replace it with this:
+
+```yaml
+modules:
+  http_2xx:
+    prober: http
+    timeout: 5s
+    http:
+      method: GET
+```
+
+With this configuration, the Blackbox Exporter will use the built-in `http_2xx` module, which periodically probes HTTP endpoints (the interval is set in your Prometheus configuration).
```

Comment:
"the built-in `http_2xx` module" -> The name `http_2xx` is actually arbitary and user-chosen in the configuration, so there's no built-in module `http_2xx`. It would be more correct to say that this configuration defines a `http_2xx` module that uses the built-in `http` prober.